log/sequencer.go

```diff
@@ -416,11 +456,15 @@ func (s Sequencer) IntegrateBatch(ctx context.Context, tree *trillian.Tree, limi
 		stageStart = s.timeSource.Now()
 
 		// Create the log root ready for signing
-		seqTreeSize.Set(float64(merkleTree.Size()), label)
+		rootHash, err := merkleTree.GetRootHash()
```

Comment:
Gotcha: `rootHash` can be nil if the tree is empty. Probably won't happen, but we should better double check it here.